assets/wizards/analytics/views/newspack-custom-events/index.js

```diff
@@ -0,0 +1,118 @@
+/* global newspack_analytics_wizard_data */
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	Button,
+	Grid,
+	Notice,
+	SectionHeader,
+	TextControl,
+	withWizardScreen,
+} from '../../../../components/src';
+
+/**
+ * Analytics Custom Events screen.
+ */
+class NewspackCustomEvents extends Component {
+	state = {
+		ga4Credendials: newspack_analytics_wizard_data.ga4_credentials,
```

Comment:
If you haven't set a measurement ID or secret before, the localized values will be `false`, and this appears as a stringified value in the fields.

<img width="2045" alt="Screen Shot 2023-07-26 at 2 22 18 PM" src="https://github.com/Automattic/newspack-plugin/assets/2230142/8ae4ff24-4a82-44da-b0e9-ce5db1b546a8">

Reply:
fixed in https://github.com/Automattic/newspack-plugin/pull/2580/commits/c77eaaa2442e5bb09f163b3c5a2c424fabf877f1